ann_cnn_training/function_training.py

```diff
@@ -28,6 +28,9 @@ def Training_ANN_CNN(
     LOSS_TRAIN = np.zeros((nepochs))
     LOSS_TEST = np.zeros((nepochs))
 
+    log = open(log_filename, "a")
```

Comment:
for logging I would really recommend using python's [logging](https://docs.python.org/3/library/logging.html) library. There is a handy tutorial [here](https://docs.python.org/3/howto/logging.html) which covers the basics.

Reply:
I would recommend you replace the `print` statements with `logging`. But it doesn't have to be done in this PR. I have raised issue #28 so we don't forget.